content/id/docs/Catalyst/kriteria-assessment-proposal.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Kriteria Penilaian Proposal"
+linkTitle: "Kriteria Penilaian Proposal"
+hide_feedback: true
+weight: 4
+description: Kriteria penilaian proposal bagi para community advisor
+---
+
+Untuk menilai sebuah proposal harus memenuhi 3 aspek, antara lain:
```

Comment:
I've changed this just to a simple body text.

Reply:
Hi, Jack, agreed. What should I do now? Commit again? Or just see your changes?

Reply:
But I haven't put ' ' or " " on the description, haha

Reply:
Hopefully, that's the last commit